pkg/virt-launcher/virtwrap/access-credentials/access_credentials_test.go

```diff
@@ -126,6 +126,87 @@ var _ = Describe("AccessCredentials", func() {
 		Expect(manager.agentSetAuthorizedKeys(domName, user, authorizedKeys)).To(Succeed())
 	})
 
+	It("should dynamically update ssh key with old qemu agent", func() {
```

Comment:
Can you also add a negative test where both sets are not available?

Reply:
done